### DISCUSSION TITLE
temporal upscaler not working in RE2

### DISCUSSION THREAD

**Discussion by LordAinzalot** [praydog/REFramework#774] [General Questions]
hey there
i´d love to use the upscaler in RE2, works perfectly in RE4 but somehow, the option is not showing up in Framework menu. Quite weird, I did it the exact same way as with RE4. Any suggestions?

thanks alot!

**Comment by praydog** [comment]
https://github.com/praydog/REFramework/actions

`pd-upscaler` branch only.

**Reply by loda98** [reply]
what do u mean by this?

**Reply by aaserra** [reply]
I understand that TemporalUpscaler is in another branch, but how can I compile it? Because I can't find a single release of that branch..

**Reply by itsmeemilio** [reply]
Rather than compiling it yourself, you can try a somewhat updated fork like this: [Upscaler Branch](https://github.com/yobson1/REFramework/releases/tag/empress). Can't promise it'll be updated/work for the version of the game you have, but it worked for me and RE Village.

**Reply by ttenor12** [reply]
That one doesn't work, unfortunately. I have not been able to find any other one built, so I guess I will need to build one myself. The issue is, I am no developer and don't know how to do so. Any idea of where to start?

**Reply by other2w** [reply]
Why was this made separately? Why not integrate the TemporalUpscaler with all of REFramework?

**Reply by praydog** [reply]
It is not separate. It's just another branch because it does not meet my quality standards for being merged into the main branch yet, because of bugs or other issues. 

The precompiled `pd-upscaler` builds are here, I'm not sure why you can't find it: https://github.com/praydog/REFramework/actions

**Reply by other2w** [reply]
I am not familiar with GitHub, and I believe many people are not either. Months ago, I struggled a lot to find this "branch" that is in another location and contains the Temporal Upscaler. People come from Nexus Mods, and there is no information there on what to download to use the Temporal Upscaler, not even an alternate link. It is not intuitive, inexperienced people cannot easily find it, and it will remain that way as long as it stays like this (unless Nexus Mods definitively use both links and explain the benefits of one over the other).

**Reply by Amentihall** [reply]
I spent 3 hours figuring this out. There are no proper guides for this. All of the ones online say to just download REFRAMEWORK with no specifics, which doesn't work for upscaling. The Nexus mod framework upscaler beta doesn't work on village, it crashes. 

The EMPRESS release works though. For GITHUB (agree very unintuitive for non devs) , go to the actions/workflow page. Go to the top right of releases, and click the little arrow on BRANCH. There you will see upscaler. Click that and voila filters latest releases with upscaler. Click topmost release and then go down and RE8.ZIP is there. 

Its crazy no one has made a proper guide for this. hope this helps someone.

**Reply by S2GB47** [reply]
@Amentihall Dude thank youuu, spent 3 hours too trying to find it and in the end I found your comment, otherwise I would have just given up.

**Reply by mnavs** [reply]
@Amentihall you are the man, just help me a lot thanks

**Reply by ZeRoFirEc** [reply]
thank you bro, really helpful

**Reply by Lando0451** [reply]
I don't understand how this isn't just part of the nightly builds or even the main releases to this day. It's so obtuse for no reason.

**Reply by ass34ass34** [reply]
Jesus F christ man, I've spent 2 days trying different version dlls back and forth, THANK YOU

**Reply by akramkhader193-hue** [reply]
guys mine says "expired" I don't know y, what can I to doo?? please helpp

**Comment by LordAinzalot** [comment]
hey there,
thanks for the quick reply but i don´t understand what you mean :D which
one is it

AND another question: how do i use dlss tweaks with yours? do they gon into
framework plugins or into main folder where re2 .exe is?

thanks alot!

Am Mi., 3. Mai 2023 um 18:11 Uhr schrieb praydog ***@***.***>:

> https://github.com/praydog/REFramework/actions
>
> pd-upscaler branch only.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/praydog/REFramework/discussions/774#discussioncomment-5793100>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7SWWL37UKUERVTIUTAKQPLXEJ7TRANCNFSM6AAAAAAXUS5LUE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Beste Grüße

Nils Heitjohann


Bodelschwinghstrasse 28
48165 Münster

Tel.: 0176 - 23805999
Email: ***@***.***

**Reply by itsmeemilio** [reply]
You can use dlss tweaks by following the tweaks tutorial for the dlss dll file, which would be stored in the main install path where the exe for the game is located.

As for installing the plugin, you'll download the nightly build here (click on the one that matches the name of your game): https://github.com/praydog/REFramework/actions/runs/7156669560

Then download the DLSS dll file here: https://www.techpowerup.com/download/nvidia-dlss-dll/

And the base upscaler plugin here: https://www.nexusmods.com/site/mods/502?tab=files

Place dinput8.dll (from 1st link), nvgx_dlss.dll (from 2nd link), and PDPerfPlugin.dll (from 3rd link) into your main game folder where the re2 exe file is located. Boot up the game and you'll see a TemporalUpscaler option like in this image:
![Screenshot 2024-01-11 112957](https://github.com/praydog/REFramework/assets/119455158/d7fd006f-18a1-4e6b-8c06-7032bd892c28)

And you can select the DLSS option you want (either DLAA, Quality, Balance, etc.)

**Reply by durrburger93** [reply]
Hey do you know why RE2 would crash on launch around the "start game" screen, when both dinput8.dll and PDPerfPlugin.dll are in RE2 game folder? Using either file alone allows the game to launch, but when they are both in the same folder it crashes every time. This doesn't happen for RE3 and RE4, only 2.

**Reply by rorunsolu** [reply]
> Hey do you know why RE2 would crash on launch around the "start game" screen, when both dinput8.dll and PDPerfPlugin.dll are in RE2 game folder? Using either file alone allows the game to launch, but when they are both in the same folder it crashes every time. This doesn't happen for RE3 and RE4, only 2.

I think the nvgx_dlss.dll (from 2nd link), and PDPerfPlugin.dll (from 3rd link) need to go into "D:\SteamLibrary\steamapps\common\RESIDENT EVIL 2  BIOHAZARD RE2\reframework\plugins" as that is what I did for RE4. I am going to test it out now with RE2. Note, for dinput8.dll (from 1st link) I will still be placing inside the game folder with the exe.

Also I used the below link to get the upscaler since the old link is expired 

https://github.com/praydog/REFramework/actions/runs/11318179052

**Reply by rorunsolu** [reply]
> > Hey do you know why RE2 would crash on launch around the "start game" screen, when both dinput8.dll and PDPerfPlugin.dll are in RE2 game folder? Using either file alone allows the game to launch, but when they are both in the same folder it crashes every time. This doesn't happen for RE3 and RE4, only 2.
> 
> I think the nvgx_dlss.dll (from 2nd link), and PDPerfPlugin.dll (from 3rd link) need to go into "D:\SteamLibrary\steamapps\common\RESIDENT EVIL 2 BIOHAZARD RE2\reframework\plugins" as that is what I did for RE4. I am going to test it out now with RE2. Note, for dinput8.dll (from 1st link) I will still be placing inside the game folder with the exe.
> 
> Also I used the below link to get the upscaler since the old link is expired
> 
> https://github.com/praydog/REFramework/actions/runs/11318179052

Update: Game doesnt start when I tried the steps emntioned in my last reply. To fix it just follow what the first guy suggested.

**Reply by Brandestoch** [reply]
> > > Hey do you know why RE2 would crash on launch around the "start game" screen, when both dinput8.dll and PDPerfPlugin.dll are in RE2 game folder? Using either file alone allows the game to launch, but when they are both in the same folder it crashes every time. This doesn't happen for RE3 and RE4, only 2.
> > 
> > 
> > I think the nvgx_dlss.dll (from 2nd link), and PDPerfPlugin.dll (from 3rd link) need to go into "D:\SteamLibrary\steamapps\common\RESIDENT EVIL 2 BIOHAZARD RE2\reframework\plugins" as that is what I did for RE4. I am going to test it out now with RE2. Note, for dinput8.dll (from 1st link) I will still be placing inside the game folder with the exe.
> > Also I used the below link to get the upscaler since the old link is expired
> > https://github.com/praydog/REFramework/actions/runs/11318179052
> 
> Update: Game doesnt start when I tried the steps emntioned in my last reply. To fix it just follow what the first guy suggested.

can you explain how you did to work pls, my game crash on start

**Reply by rorunsolu** [reply]
It's the same method that the 1st guy in the thread mentioned in (https://github.com/praydog/REFramework/discussions/774#discussioncomment-8098234).

FYI, the new link "**Nightly Build**" is (https://github.com/praydog/REFramework-nightly/releases/). I'm not sure if you are used to modding games so I'll ask a question or two about your current setup.

Are you running the game in **Dx12** mode or **Dx11**?

If it is **Dx11** then the game name in Steam will look like the image below (it will directly say dx11_non-rt in the name alongisde RE2). However if it is running the game in the **default mode** aka **Dx12** then it will just say "resident evil 2".

Once you have confirmed which DX version you are running the game on then you will need to pick the corrosponding file to download in the "**Nightly Build**" link I mentioned before (https://github.com/praydog/REFramework-nightly/releases/).

Note: The link will mention something about "non-rt" version which refers to **Dx11**. Earlier you had to confirm if you are using Dx11 or Dx12 to run the game. If it is Dx11 then you need to download the "**RE2_TDB66.zip**" file as indicated by the "If you want builds that work on the old non-RT versions, download the versions that have "TDB" in them." found at the start of the webpage of that which is again (https://github.com/praydog/REFramework-nightly/releases/).

if you are running the game in Dx12 then you will need to download the file called "RE2.zip".

Now follow the instructions from the 1st guy in the thread that I mentioned before. You can just scroll up tyo find it but here is the link too (https://github.com/praydog/REFramework/discussions/774#discussioncomment-8098234).


![image](https://github.com/user-attachments/assets/2bc18d93-d709-4fbb-bd92-b9163ec72911)

**Reply by Brandestoch** [reply]
> It's the same method that the 1st guy in the thread mentioned in ([#774 (reply in thread)](https://github.com/praydog/REFramework/discussions/774#discussioncomment-8098234)).
> 
> FYI, the new link "**Nightly Build**" is (https://github.com/praydog/REFramework-nightly/releases/). I'm not sure if you are used to modding games so I'll ask a question or two about your current setup.
> 
> Are you running the game in **Dx12** mode or **Dx11**?
> 
> If it is **Dx11** then the game name in Steam will look like the image below (it will directly say dx11_non-rt in the name alongisde RE2). However if it is running the game in the **default mode** aka **Dx12** then it will just say "resident evil 2".
> 
> Once you have confirmed which DX version you are running the game on then you will need to pick the corrosponding file to download in the "**Nightly Build**" link I mentioned before (https://github.com/praydog/REFramework-nightly/releases/).
> 
> Note: The link will mention something about "non-rt" version which refers to **Dx11**. Earlier you had to confirm if you are using Dx11 or Dx12 to run the game. If it is Dx11 then you need to download the "**RE2_TDB66.zip**" file as indicated by the "If you want builds that work on the old non-RT versions, download the versions that have "TDB" in them." found at the start of the webpage of that which is again (https://github.com/praydog/REFramework-nightly/releases/).
> 
> if you are running the game in Dx12 then you will need to download the file called "RE2.zip".
> 
> Now follow the instructions from the 1st guy in the thread that I mentioned before. You can just scroll up tyo find it but here is the link too ([#774 (reply in thread)](https://github.com/praydog/REFramework/discussions/774#discussioncomment-8098234)).
> 
> ![image](https://private-user-images.githubusercontent.com/145875286/382266605-2bc18d93-d709-4fbb-bd92-b9163ec72911.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NjY4ODEsIm5iZiI6MTczMDQ2NjU4MSwicGF0aCI6Ii8xNDU4NzUyODYvMzgyMjY2NjA1LTJiYzE4ZDkzLWQ3MDktNGZiYi1iZDkyLWI5MTYzZWM3MjkxMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxMzA5NDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ODM1Y2Q0OWE0MmY2NzFhMDgzY2Q1NDJkODJhNGI5OTE5NzgwYTI1ODhhOTJhYmU0ZWE0NjlmODc4YjEwM2IzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.lrWsvFBgxwNHAlnZvPj0wZ8g9EaQEOAK-N9ztdACj68)

tks for kind reply.
So i have download the nightly releases, both of them, the framework loads ok, but the upscaler options doesnt show.
So i downloaded the pd-upscaler version, Now the upscaler option shows, but the game crahs soon as load the framework.
if i delet the nvngx_dlss.dll, the game doenst crash anymore, but the upscale shows a message error e then doesnt work.

**Reply by rorunsolu** [reply]
where have you placed the each of the files?
which game version are you running?

**Reply by Brandestoch** [reply]
> where have you placed the each of the files? which game version are you running?

I placed in the root folder were the re2.exe is
RT version

**Reply by rorunsolu** [reply]
can you provide a screenshot of your entire re2 folder (where the .exe is) so i can see?

**Reply by Brandestoch** [reply]
> can you provide a screenshot of your entire re2 folder (where the .exe is) so i can see?

![1](https://github.com/user-attachments/assets/457216d9-4693-4f70-a9f1-47755b602be4)

**Reply by rorunsolu** [reply]
The game probably isn't working because you're trying to use reshade mods.

Reshade doesn't work in dx12 which is what you are using.

Try uninstalling reshade from the game by going to reshade exe and uninstallign from there.

I also had the same issue which is why I switched the game to run on Dx11.

I played the game with DX11, Reshade and Reframework for Ultrawide (21:9) fix & the upscaler (dlss).

**Reply by Brandestoch** [reply]
> The game probably isn't working because you're trying to use reshade mods.
> 
> Reshade doesn't work in dx12 which is what you are using.
> 
> Try uninstalling reshade from the game by going to reshade exe and uninstallign from there.
> 
> I also had the same issue which is why I switched the game to run on Dx11.
> 
> I played the game with DX11, Reshade and Reframework for Ultrawide (21:9) fix & the upscaler (dlss).

i tried a freh instal to. without mods an reshade, doesnt work.
how do i change to dx11?

**Reply by rorunsolu** [reply]
go to steam and right click re2 and go to the location in the image below and click the beta participation dropdown list and select the same as in the screenshot. You won't be able to use raytracing if you activate this though. Also it will start installing the DX11 version and if you want to go back to DX12 it will install DX12 again. I think it reinstalls the entire game each time

![image](https://github.com/user-attachments/assets/9ae06289-4fd9-4919-b16d-751197a4ca09)

**Reply by Brandestoch** [reply]
> go to steam and right click re2 and go to the location in the image below and click the beta participation dropdown list and select the same as in the screenshot. You won't be able to use raytracing if you activate this though. Also it will start installing the DX11 version and if you want to go back to DX12 it will install DX12 again. I think it reinstalls the entire game each time
> 
> ![image](https://private-user-images.githubusercontent.com/145875286/382327234-9ae06289-4fd9-4919-b16d-751197a4ca09.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0NzY5NTgsIm5iZiI6MTczMDQ3NjY1OCwicGF0aCI6Ii8xNDU4NzUyODYvMzgyMzI3MjM0LTlhZTA2Mjg5LTRmZDktNDkxOS1iMTZkLTc1MTE5N2E0Y2EwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxNTU3MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDIwZThlMjk4ZDUxNzIxZTE4MjcwOTQyZGYzZGQxOTdmMzg2NWE3MWI2MDI1ZTdjNDQxOTQ0YjM2NzJhZWU2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.TeUkkEWpcoXu2356lFNkeHOkznCqocS8g8DLVsOKC-g)

ok il try. tks for your time helping me.

**Reply by rorunsolu** [reply]
no worries mate

**Comment by TaurusHarva** [comment]
I can't get it to work in re2 Remake & Devil May Cry 5 either. It wouldn't be bad if a short video tutorial was made to clear up any doubts.

**Reply by praydog** [reply]
Does not work in DMC5 yet. Also doesn't work in RE2 if you are using the non-rt (DX11) branch.

https://github.com/praydog/REFramework/actions

Download `pd-upscaler` builds from there if you are downloading the wrong one.

**Reply by savi4587** [reply]
> Does not work in DMC5 yet. Also doesn't work in RE2 if you are using the non-rt (DX11) branch.
> 
> https://github.com/praydog/REFramework/actions
> 
> Download `pd-upscaler` builds from there if you are downloading the wrong one.

 have you ever worked on a dx11 version at all? just wondering. even if it barely works, it's fine cause I'll take it over the game's built in TAA. it is horrendous. and all the good mods are on the non-RT version which came out 3 years after the game's release (doesn't make sense to port the mods into rt when the game is 3 years old, does it?)

**Comment by TaurusHarva** [comment]
With those files it now works in RE2 Remake, 1440p, Ray Tracing on High and maximum graphics I can get 60fps with DLSS on balanced. amazing c:

RTX 2070
I9 9900k
32GB RAM

![Resident Evil 2   Biohazard 2 Screenshot 2023 12 18 - 15 03 46 35](https://github.com/praydog/REFramework/assets/43914977/03e9b395-8be2-4cf9-bf9c-ed2ec59cec66)

**Reply by makemeone-prog** [reply]
hey were you playing the non rtx (dx11) edition? im asking since praydog said that the non rtx framework does not support dlss

**Comment by Eddliz** [comment]
I download the v1.5.2 of the REFramvework and still can't see the TemporalUpsacler option
![image](https://github.com/praydog/REFramework/assets/50850832/fe7a300c-b27b-4979-ba2d-f0dc9b6ef3ac)
![image](https://github.com/praydog/REFramework/assets/50850832/70b45582-c895-466c-afc9-2c724b55a0e4)

this are the dlss and UpscallerBasePlugin that I download
![image](https://github.com/praydog/REFramework/assets/50850832/b3345732-fd82-4652-8286-5ca86844d7f4)

**Reply by itsmeemilio** [reply]
Check out the writeup I did on how to enable it. Those instructions were working as of a couple days ago:

https://github.com/praydog/REFramework/discussions/774#discussioncomment-8098234

**Reply by Eddliz** [reply]
thanks now it's working

**Reply by abragagn** [reply]
@Eddliz What did you do wrong? I'm following @itsmeemilio instructions and the temporal upscaled does not show up in reframework.

![RESIDENT EVIL 2 16_01_2024 17_45_48](https://github.com/praydog/REFramework/assets/17766798/e841ac7a-c1bd-46cb-afb1-d2773f748a07)

All the three .dll are in the game folder:

![image](https://github.com/praydog/REFramework/assets/17766798/d427a40a-33e6-4fdb-a884-b78bc1422d3a)

I'm launching the game from the PC xbox (game pass) app, if it is relevant.

**Reply by Eddliz** [reply]
well for me I have the steam version of the game, not sure if it the same game version

but what it looks like bad for me was the REFramework dinput8.dll

so I press this link
![image](https://github.com/praydog/REFramework/assets/50850832/77d55f41-b920-4366-a03b-a0f7e4decc59)
 then download this file
![image](https://github.com/praydog/REFramework/assets/50850832/050fda06-a1b3-4023-a7f8-c8bbbe51da50)
and extract the dinput8.dll to my game folder

I delete the previous dinput8.dll and the reframework folder that automaticly create when you open the game

**Reply by abragagn** [reply]
Thanks. Unfortunately the temporal upscaler menu still does not appear. Maybe it is an issue with the night build I'm using for the Xbox PC installation (@praydog).

**Reply by praydog** [reply]
The nightly build is not what you want. You need to get the last 'pd-upscaler' build from the [Actions page](https://github.com/praydog/REFramework/actions)

**Reply by arfiandi16** [reply]
Reframework Upscaler Beta doesn't work on RE4, the game doesn't even opened when I added dinput8.dll in the game directory, but when I deleted it, the game opened without reframework in it.

**Reply by arfiandi16** [reply]
Oh wait now it works, I downloaded from here https://github.com/praydog/REFramework/actions/runs/7156669560

![image](https://github.com/praydog/REFramework/assets/26373000/0190cbc8-d260-48c9-99a9-5d59445400e5)

**Reply by ferrufinosarce-art** [reply]
Download the Upscaler Branch tab to get the TemporaryUpsacaler, then put the UpscalerBasePlugin 1.1.2 on it and you can use the Xess or FSR 2.0.
It worked for me.

https://github.com/praydog/REFramework/actions/runs/22970524660

**Comment by durrburger93** [comment]
Thanks for these guides guys, but after putting both the dinput8.dll from the RE2 on this link: https://github.com/praydog/REFramework/actions/runs/7544051405, AND the PDPerfPlugin.dll from base upscaler plugin, the game crashes to desktop after the "start game" screen. 

Without the PDPerfPlugin.dll, the framework UI appears ingame along with the temporal upscale section, and the game does work, but there are no options inside the temp upscale section. 

Does anyone know why this would be causing the crash? It's the DX12 version of RE2.

**Reply by nildooliveira21** [reply]
tbm estou com o msm problema

**Comment by RoblozScriptMan** [comment]
I fixed my issue by deleting reframework and the old input.dll and the pd upscaler dll and replacing it with the one from 1.1.12 then downloaded https://github.com/praydog/REFramework/actions/runs/11770014195 and put the input dll into the re folder

**Comment by RiasatSalminSami** [comment]
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/2aab6284-7f48-445f-966a-ab134511a57d" />

 I can't seem to click any of these to download anything. None of them are clickable on the pd branch. Need some help.

**Reply by rorunsolu** [reply]
You're on the wrong page I believe. You need to go back to the main page for the repo and click the releases tab

**Reply by RiasatSalminSami** [reply]
> You're on the wrong page I believe. You need to go back to the main page for the repo and click the releases tab

hmm, but on release tab, I can't sort out the pd-upscaler branch. And the newer branches don't work.

**Reply by rorunsolu** [reply]
> > You're on the wrong page I believe. You need to go back to the main page for the repo and click the releases tab
> 
> hmm, but on release tab, I can't sort out the pd-upscaler branch. And the newer branches don't work.

I'm not on my PC at the moment but I probably mentioned where to find the one you need and where in one of my earlier replies in this thread. If I have time later I can help further

**Reply by makemeone-prog** [reply]
*sigh* same, i am not sure but i don't think github was created with the intention of typical end users ease of access, i noticed many of the releases are highly nuanced and even then very difficult to navigate and find the desired "repo?" 

either way the general releases aren't compatible alot of the time... ( alot of trouble trying to play resident evil 7 vr)( and loads of trouble locating the proper pdupscaler branch for use with re2-re4 upscaler mods)

**Reply by RiasatSalminSami** [reply]
> > > You're on the wrong page I believe. You need to go back to the main page for the repo and click the releases tab
> > 
> > 
> > hmm, but on release tab, I can't sort out the pd-upscaler branch. And the newer branches don't work.
> 
> I'm not on my PC at the moment but I probably mentioned where to find the one you need and where in one of my earlier replies in this thread. If I have time later I can help further

yeah, I found the place that you pointed out. The problem is, there seems to be no way to download anything there as I've shown on the screenshot I posted earlier. Nothing is clickable there.

**Comment by mihkeh** [comment]
o que eu fiz de errado? tenho a opção do temporalupscaler mas não consigo configurar diz que é imcompativel

<img width="315" height="827" alt="Captura de tela 2025-12-17 154605" src="https://github.com/user-attachments/assets/f6014f41-88f4-40ce-bd1a-f5e2013f7219" />

**Comment by RiasatSalminSami** [comment]
All the links have expired. The pd branch versions are no longer downloadable!!

**Reply by Mephilas** [reply]
https://github.com/praydog/REFramework/actions/runs/20196406549
just click your game name like RE4

**Reply by RiasatSalminSami** [reply]
> https://github.com/praydog/REFramework/actions/runs/20196406549 just click your game name like RE4

Much appreciated. Thanks a lot.

**Reply by Sanselw** [reply]
> https://github.com/praydog/REFramework/actions/runs/20196406549 just click your game name like RE4

It's expired again. Now what?? :(
